.github/workflows/build_docs.yaml

```diff
@@ -30,6 +30,7 @@ jobs:
         run: |
           mamba install --quiet --file=requirements.txt          
           mamba install --quiet pip
+          pip install lsst.resources
```

Comment:
We should get this into conda-forge ... 
but I'm wondering if we can't add it to the requirements.txt file anyway? 
If it's not necessarily for most uses, but only for the ones we're in control of for now, this is probably fine.

Reply:
Currently, requirments.txt is just a simple text list of packages, and I don't think this format supports pip/pypi packages. Such packages are supported by the conda's yaml format for specifying environments, and perhaps we should move to that format, but maybe that should be done as a separate issue (and corresponding PR).